src/main/java/competition/subsystems/drive/commands/vision_path/PathDriveToLocationCommand.java

```diff
@@ -0,0 +1,134 @@
+package competition.subsystems.drive.commands.vision_path;
+
+import competition.subsystems.oracle.ReefRoutingCircle;
+import competition.subsystems.pose.Landmarks;
+import competition.subsystems.pose.PoseSubsystem;
+import competition.subsystems.vision.AprilTagVisionSubsystemExtended;
+import competition.subsystems.vision.CoprocessorCommunicationSubsystem;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import org.kobe.xbot.JClient.XTablesClient;
+import org.kobe.xbot.Utilities.Entities.XTableValues;
+import org.kobe.xbot.Utilities.VisionCoprocessorCommander;
+import xbot.common.logging.RobotAssertionManager;
+import xbot.common.properties.PropertyFactory;
+import xbot.common.subsystems.drive.BaseSwerveDriveSubsystem;
+import xbot.common.subsystems.drive.control_logic.HeadingModule;
+import xbot.common.trajectory.XbotSwervePoint;
+
+import javax.inject.Inject;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class PathDriveToLocationCommand extends SwerveBezierTrajectoryBase {
+    Pose2d target;
+
+    private final VisionCoprocessorCommander commander;
+
+    private static boolean useBackupPointToPoint = false;
+
+    private XTableValues.TraversalOptions traversalOptions;
+    private int safeInches = 10;
```

Comment:
⭐ Consider using a `Distance` value so the units are baked in. Also generally wouldn't suggest `int` for a value like this that you could want to have a decimal component to

```suggestion
    private Distance safeDistance = Inches.of(10);